test/no-single-promise-in-promise-methods.mjs

```diff
@@ -6,7 +6,7 @@ const {test} = getTester(import.meta);
 test.snapshot({
 	valid: [
 		'Promise.all([promise, anotherPromise])',
-		'Promise.all(notArrayExpression)',
+		'Promise.all(arrayVariable)',
```

Comment:
Isn't the old name more clear?

Reply:
I felt it was unclear. It could alternatively be `notArrayLiteral`.

Reply:
It's done.